### PR TITLE
Add drivers to dataloader

### PR DESCRIPTION
**Pull request by Aebel-Shajan**

### CODE REVIEW

**Review by Tommybui28 — APPROVED** [review]
impeccable

**Review by sahilpatel1906 — APPROVED** [review]

File: capstone_backend/src/main/java/com/example/Capstone_Backend/models/Driver.java
Comment (Aebel-Shajan) [inline capstone_backend/src/main/java/com/example/Capstone_Backend/models/Driver.java:1]: Shoulda been string, good call

**Review by Jean-005 — APPROVED** [review]
Excellent work as always Sahil :)!

**Review by ney601 — APPROVED** [review]
Happy with the changes

**Pull request merged**